browser/net/brave_network_audit_allowed_lists.h

```diff
@@ -24,26 +24,29 @@ inline constexpr auto kAllowedUrlProtocols =
         "blob",
     });
 
-// Before adding to this list, get approval from the security team.
-inline constexpr auto kAllowedUrlPrefixes = std::to_array<std::string_view>({
+// Before adding to the lists below, get approval from the security team.
+
+// List of URL prefixes that are used in functionality not controlled via
+// admin policies.
+inline constexpr auto kBaseAllowedUrlPrefixes = std::to_array<std::string_view>({
     // allowed because it 307's to https://componentupdater.brave.com
     "https://componentupdater.brave.com/service/update2",
     "https://crxdownload.brave.com/crx/blobs/",
 
     // Omaha/Sparkle
     "https://updates.bravesoftware.com/",
 
-    // stats/referrals
-    "https://usage-ping.brave.com/",
-
     // needed for DoH on Mac build machines
     "https://dns.google/dns-query",
 
     // needed for DoH on Mac build machines
     "https://chrome.cloudflare-dns.com/dns-query",
 
-    // for fetching tor client updater component
-    "https://tor.bravesoftware.com/",
+    // brave A/B testing
+    "https://variations.brave.com/seed",
+
+    // Brave's Privacy-focused CDN
+    "https://pcdn.brave.com/",
```

Comment:
This one is only used by Brave News as far as I know, so it could be in the bottom array with `brave-today-cdn.brave.com`?

Reply:
Ah right, we use it for publisher images IIRC.

Do we enable Rewards in the network audit test?